packages/amplify-codegen-e2e-tests/src/__tests__/graphql-documents-generator.test.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  initProjectWithProfile,
+  DEFAULT_JS_CONFIG,
+  addApiWithBlankSchemaAndConflictDetection,
+  updateApiSchemaWithText,
+  craInstall,
+  craBuild,
+  amplifyPush,
+  cypressRun,
+  deleteProject
+} from '@aws-amplify/amplify-codegen-e2e-core';
+import { readdirSync, rmSync } from 'fs';
+import { readFileSync } from 'fs-extra';
+import path from 'path';
+
+describe('GraphQL documents generator e2e tests', () => {
+  let apiName: string;
+  const projectRoot = path.resolve('test-apps', 'docsgen-react-app');
+  const config = DEFAULT_JS_CONFIG;
+
+  beforeAll(async () => {
+    await initProjectWithProfile(projectRoot, { ...config });
+    await addApiWithBlankSchemaAndConflictDetection(projectRoot);
```

Comment:
Is datastore required for this e2e test?

Reply:
Not really. Removed it to non-datastore API.